src/unstable-temp/DownloadImageModal/ImageForm.tsx

```diff
@@ -154,7 +156,22 @@ export const ImageForm = ({
 	);
 
 	const actions: ModalAction[] = [
-		...(modalActions ?? []),
+		{
+			plain: true,
+			onClick: (event) => flashWithEtcher(event, downloadOptions),
```

Comment:
What happens if I select a docker only DT (eg `aarc64` on prod)?
Is the button clickable/disabled?

Reply:
screenshots added